components/accelerator/nvidia/ecc/component_output.go

```diff
@@ -13,19 +13,20 @@ import (
 
 func ToOutput(i *nvidia_query.Output) *Output {
 	o := &Output{}
-	for _, g := range i.SMI.GPUs {
-		if g.ECCErrors == nil {
-			continue
-		}
+	if i != nil && i.SMI == nil {
+		for _, g := range i.SMI.GPUs {
+			if g.ECCErrors == nil {
+				continue
+			}
 
-		o.ErrorCountsSMI = append(o.ErrorCountsSMI, *g.ECCErrors)
+			o.ErrorCountsSMI = append(o.ErrorCountsSMI, *g.ECCErrors)
```

Comment:
See https://github.com/leptonai/gpud/pull/83. Cherry-picked this change to a separate PR.